package com.project.najdiprevoz.exceptions

class InvalidUserIdException(id: Long) : RuntimeException("EXCEPTION_USER_NOT_FOUND")